app/code/Magento/MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js

```diff
@@ -289,6 +291,13 @@ define([
             }
         },
 
+        /**
+         * Action to close the context menu in media gallery.
+         */
+        closeContextMenu: function () {
+            $(this.gridSelector).click();
```

Comment:
I think this fix is risky and may interfere with other functionalities. I'll check if there are more localized approaches